specification/connector/secret_post/ConnectorSecretPostRequest.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+
+/**
+ * Creates a secret for a Connector.
+ *
+ * @rest_spec_name connector.secret_post
+ * @availability stack stability=experimental visibility=private
+ */
+export interface Request extends RequestBase {
+  urls: [
+    {
+      path: '/_connector/_secret'
+      methods: ['POST']
+    }
+  ]
+  body: {
+    value: string
+  }
+}
```

Comment:
https://github.com/elastic/elasticsearch/blob/6140b020c8462432ac9dcf84f48b3d77b4ff9650/x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/secrets/action/RestPostConnectorSecretAction.java#L29-L44

https://github.com/elastic/elasticsearch/blob/6140b020c8462432ac9dcf84f48b3d77b4ff9650/x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/secrets/action/PostConnectorSecretRequest.java#L35-L42

Should value really be optional?